HaxeManual/02-types.tex

```diff
@@ -734,15 +734,15 @@ \subsection{Any type}
 \label{types-dynamic-any}
 
 \type{Any} is a type that is compatible with any other in both ways. 
-It serves one purpose - to hold values of any type, but to actually use that values, an explicit casting is required. That way the code doesn't suddently become dynamically typed and we keep all the static typing goodness, like advanced type system features and optimizations.
+It serves one purpose - to hold values of any type but to actually use that values, explicit casting is required. That way the code doesn't suddenly become dynamically typed and we keep all the static typing goodness, like advanced type system features and optimizations.
```

Comment:
The comma after `type` should remain – those are two independent clauses. `that values` -> `those values`.